pkgs/applications/window-managers/i3/lock-fancy.nix

```diff
@@ -3,32 +3,34 @@
 }:
 
 stdenv.mkDerivation rec {
-  rev = "3734fba160166006521e513f5734eb76ac6aa48f";
-  name = "i3lock-fancy-unstable-2017-12-14_rev${builtins.substring 0 7 rev}";
+  rev = "f2e9bdd1d27e761f0dca75b89329865545351d70";
+  name = "i3lock-fancy-unstable-${version}";
+  version = "2018-08-27_rev${builtins.substring 0 7 rev}";
```

Comment:
Can we fix this to conform to the normal unstable scheme? The final package name should be "i3lock-fancy-unstable-2018-08-27"